himsog/models.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+from django.template.defaultfilters import slugify
+
+
+class Category(models.Model):
+
+    name = models.CharField(max_length=256)
```

Comment:
Hindi natin gagawing defined 'yung categories? Lagyan ng list of choices sa param ng name?